src/commodore/readme.md

```diff
@@ -12,6 +12,8 @@ The manager will optionally try to fetch cluster info from Lieutenant.
 Data returned from Lieutenant will be provided to `commodore inventory show` to infer more accurate component and package URLs (e.g. if an URL is overridden for a distribution).
 This feature is only available for tenant repositories, and only if configuration parameters `lieutenantURL` and `lieutenantToken` are set.
 
+For Commodore components hosted on GitHub, the manager will use GitHub releases as the data source for detecting new versions. For all other git hosts, it will use git refs, detecting versions from among the repository's git tags and git heads.
```

Comment:
Please try to keep documentation at one sentence per line. That makes reviewing changes easier.

```suggestion
For Commodore components hosted on GitHub (cf. extra parameter `githubBaseUrl`), the manager will use GitHub releases as the data source for detecting new versions.
For all other Git hosting platforms, it will use git refs, detecting versions from among the repository's git tags and git heads.
```